AlphaWallet/Common/Views/ViewModels/TokenCardRowViewModelProtocol.swift

```diff
@@ -46,7 +46,7 @@ extension TokenCardRowViewModelProtocol {
     }
 
     var iconsColor: UIColor {
-        return Configuration.Color.Semantic.defaultHeadlineText
+        return Configuration.Color.Semantic.symbol
```

Comment:
Is this symbol as in a token (eg. ERC20's) symbol? Otherwise maybe `.icon` would be better?

Reply:
There is already a different color named icon.

Reply:
Sure